src/libraries/Microsoft.Extensions.Configuration.Binder/gen/ConfigurationBindingGenerator.cs

```diff
@@ -73,7 +69,11 @@ private sealed record CompilationData
 
             public CompilationData(CSharpCompilation compilation)
             {
-                LanguageVersionIsSupported = compilation.LanguageVersion >= LanguageVersion.CSharp11;
+                // We don't have a CSharp21 value available yet. Polyfill the value here for forward compat, rather than use the LangugeVersion.Preview enum value.
```

Comment:
Did you mean CSharp12?

Reply:
Yeah, will fix if this feature stays in the release.